languages/rust/exercises/concept/numbers/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+pub fn production_rate_per_hour(speed: u8) -> f64 {
+    unimplemented!("calculate hourly production rate at speed: {}", speed)
```

Comment:
Should we use the `todo!` macro?

Reply:
Could, but the existing exercises all use unimplemented too. They both make sense in the context.